neural_modelling/src/spike_source/poisson/spike_source_poisson.c

```diff
@@ -352,28 +342,22 @@ void resume_callback() {
 //! \brief stores the poisson parameters back into sdram for reading by the
 //! host when needed
 //! \return None
-bool store_poisson_parameters(){
+bool store_poisson_parameters() {
     log_info("stored_parameters: starting");
 
     // Get the address this core's DTCM data starts at from SRAM
     address_t address = data_specification_get_data_address();
     address = data_specification_get_region(POISSON_PARAMS, address);
-    uint32_t seed_size = sizeof(mars_kiss64_seed_t) / sizeof(uint32_t);
 
     // Copy the current seed
-    memcpy(&address[PARAMETER_SEED_START_POSITION], spike_source_seed,
-    		seed_size);
-
-    // Get number of spike sources
-    num_spike_sources = address[PARAMETER_SEED_START_POSITION + seed_size];
-    log_info("\t spike sources = %u", num_spike_sources);
+    parameters.spike_source_seed = spike_source_seed;
```

Comment:
I'm not sure that this is correct; it might be better off being deleted if the seed is being stored directly in the `parameters`.